CMakeLists.txt

```diff
@@ -75,3 +75,19 @@ endif()
 if (${BUILD_TESTS})
 	add_subdirectory (tests)
 endif ()
+
+set(INCLUDE_INSTALL_DIR include/)
+set(LIB_INSTALL_DIR lib/)
+set(LIBJWT_VERSION 1.13.1)
```

Comment:
Eh. I'm going to forget to update this along with the `configure.ac` for every release. Is there a way to extract this from `configure.ac` instead, so I only have to have it in one place and avoid possible skew?

Reply:
puh, yah, may be possible by grepping. gonna try tomorrow

Reply:
ok, that should do it